backend/tracim_backend/lib/core/content.py

```diff
@@ -2490,3 +2503,55 @@ def set_content_namespace(
         )
 
         return content
+
+    def get_content_property(
+        self,
+        property_list: typing.List[QueryableAttribute],
+        content_id: int = None,
+        revision_id: int = None,
+    ) -> typing.Tuple:
+        """
+        Return the specified properties of a revision.
+        - If revision_id is provided, return the properties for that revision.
+        - If only content_id is provided, return the properties of the last revision for that content.
+        - If both are provided, ensure the revision belongs to the content.
+
+        Args:
+            property_list (List[QueryableAttribute]): List of SQLAlchemy column attributes to retrieve
+            content_id (int, optional): ID of the content
+            revision_id (int, optional): ID of the specific revision
+
+        Returns:
+            tuple: Values of requested properties in same order as property_list
+
+        Raises:
+            ValueError: If neither content_id nor revision_id is provided
+            ContentRevisionNotFound: If the specified revision is not found
+            ContentNotFound: If the specified content is not found
+        """
+        if revision_id is None and content_id is None:
```

Comment:
I suggest to use assertion here: We dont want to manage clearly a case that is not a use case.

```
assert content_id or revision_id, "Either content_id or revision_id must be provided"
```